php/php-pcov/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               php 1.1
+
+name                    php-pcov
+version                 1.0.11
+categories-append       devel
+platforms               darwin
```

Comment:
this line is set by default since MacPorts v2.7 so it can be removed

Reply:
Aha, great! Sorry for my ignorance! Amended in fixup commit.